cloud_governance/policy/policy_operations/aws/tag_user/tag_iam_user.py

```diff
@@ -89,7 +91,7 @@ def generate_user_csv(self):
         tag_values = {}
         for user in users:
             user_name = user.get('UserName')
-            if '-' not in user_name:
+            if user_name.count('-') <= 3:
```

Comment:
can we based on count '-' number ?

Reply:
Not actually, but we need to do it now. As we are moving to multiple accounts.

Reply:
Are you sure it will not affect backward compatibility ?

Reply:
yes, it won't